pkg/controller/datavolume-controller.go

```diff
@@ -300,6 +304,24 @@ func (r *DatavolumeReconciler) sourceInUse(dv *cdiv1.DataVolume) (bool, error) {
 	return len(pods) > 0, nil
 }
 
+func (r *DatavolumeReconciler) getStorageClassBindingMode(dataVolume *cdiv1.DataVolume) (storagev1.VolumeBindingMode, error) {
+	var targetPvcStorageClass *storagev1.StorageClass
+	targetPvcStorageClassName := dataVolume.Spec.PVC.StorageClassName
+
+	// Handle unspecified storage class name, fallback to default storage class
+	targetPvcStorageClass, err := r.getStorageClassByName(targetPvcStorageClassName)
+	if err != nil {
+		return "", err
```

Comment:
would returning `*storagev1.VolumeBindingMode`be nicer?

Reply:
yes, 
no value == nil